plugin.xml

```diff
@@ -106,10 +106,10 @@
 
         <config-file target="AndroidManifest.xml" parent="/manifest/application">
             <activity
-                android:name=".wxapi.WXEntryActivity"
+                android:name="${applicationId}.wxapi.WXEntryActivity"
```

Comment:
I happened to notice that just prefixing with `.wxapi` here results in the following entry in the merged manifest:


```xml
        <activity
            android:name="capacitor.cordova.android.plugins.wxapi.WXEntryActivity"
            android:exported="true"
            android:label="@string/launcher_name"
            android:launchMode="singleTask"
            android:taskAffinity="com.smartly.hybrid"
            android:theme="@android:style/Theme.Translucent.NoTitleBar" />
```

Which is clearly wrong. This is happening because this plugin actually doesn't end up in our Capacitor app's module, it ends up in a capacitor-cordova-android-plugins` module that just gets merged with our app at build time. But the net effect is that `<activity>` declaration is pointing to a non-existent activity...